src/components/Tabs/TabButton.svelte

```diff
@@ -9,6 +9,7 @@
   export let to = "";
   export let selected = "";
   export let color = "primary";
+  export let notselectedcolor = "white";
```

Comment:
Let's keep prop naming consistent. `notSelectedColor`?

Reply:
Yes sure!
Will you make the modifications or should I update de PR?

Reply:
Please post an update, I'd probably need to fork your fork otherwise.

Reply:
Updated in the new commit.

Reply:
Thanks!

Reply:
My pleasure, I love this project!
Any idea when you are gonna push these changes to npm?

Reply:
Let me do a minor release.

Reply:
Thanks!

Reply:
Let me try to fix treeviews in the same release first though.

Reply:
Awesome!